target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -93,9 +93,6 @@ glinet,gl-ar150)
 	ucidef_set_led_netdev "wan" "WAN" "$boardname:green:wan" "eth1"
 	ucidef_set_led_switch "lan" "LAN" "$boardname:green:lan" "switch0" "0x02"
 	;;
-glinet,gl-ar300m-nor)
-	ucidef_set_led_netdev "lan" "LAN" "gl-ar300m:green:lan" "eth0"
-	;;
```

Comment:
What's the reason for removing this?

Reply:
Redundant and in keeping with similar core-dev requests.
`dts/qca9531_glinet_gl-ar300m.dtsi`
```
                led_wlan: wlan {
                        label = "gl-ar300m:red:wlan";
                        gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
                        linux,default-trigger = "phy0tpt";
                };
```
(Or https://github.com/openwrt/openwrt/blob/master/target/linux/ath79/dts/qca9531_glinet_gl-ar300m.dtsi#L56)

Reply:
Moved to target/linux/ath79/nand/base-files/etc/board.d/01_leds as the glinet,gl-ar300m-nor is on the NAND target, not the generic target.